src/current/_includes/v24.3/resilience/dr-feature-table.md

```diff
@@ -0,0 +1,64 @@
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Point-in-time backup & restore</th>
+    <th>Physical cluster replication (asynchronous)</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
+    </td>
+    <td>>=5 minutes</td>
+    <td>10s of seconds</a></td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RTO</b>
+    </td>
+    <td>Minutes to hours, depending on data size and number of nodes</td>
+    <td>Seconds to minutes, depending on cluster size</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Write latency</b>
+    </td>
+    <td>No impact</td>
+    <td>No impact</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Recovery</b>
+    </td>
+    <td>Manual restore</td>
+    <td>Manual failover</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Hardware cost</b>
+    </td>
+    <td>1x + S3-like storage & networking</td>
+    <td>2x + networking between datacenters</td>
+  </tr>
```

Comment:
I don't think the 1x and 2x are clear here. Perhaps we need an explanation as a footnote at the bottom of the table?

Reply:
Maybe we say "1x CRDB cluster cost" here we are referring to the fact that in PCR you need to have two clusters running

also instead of S3-like storage,  could we say external object storage?

Reply:
I have deleted the hardware line items for now. It seemed like there were differing opinions — but I can add this back in, if we want.